test/e2e/ipsec_test.go

```diff
@@ -129,11 +135,11 @@ func testIPSecTunnelConnectivity(t *testing.T, data *TestData, certAuth bool) {
 	}
 	podInfos, deletePods := createPodsOnDifferentNodes(t, data, data.testNamespace, tag)
 	defer deletePods()
+	t.Logf("Executing ping tests between Pods across Nodes: '%v' <-> '%v'", podInfos[0].nodeName, podInfos[1].nodeName)
 	data.runPingMesh(t, podInfos[:2], agnhostContainerName)
 
-	// We know that testPodConnectivityDifferentNodes always creates a Pod on Node 0 for the
-	// inter-Node ping test.
-	nodeName := nodeName(0)
+	// Check that there is at least one 'up' Security Association on the Node
+	nodeName := podInfos[0].nodeName
```

Comment:
It seems that this is the actual bug fix. Is this addressing https://github.com/antrea-io/antrea/issues/4477#issuecomment-1499414201? If yes, I think we could consider closing the issue when we merge this

Reply:
Yes, this is the actual fix. I have included Fixes: #4447 in the commit message but forgot to add it to the PR description.